Content.Server/Botany/SeedPrototype.cs

```diff
@@ -293,4 +310,82 @@ public SeedData Clone()
 
         return newSeed;
     }
+
+
+    /// <summary>
+    /// Handles copying most species defining data from 'other' to this seed while keeping the accumulated mutations intact.
+    /// </summary>
+    public SeedData SpeciesChange(SeedData other)
+    {
+        var newSeed = new SeedData
+        {
+            Name = other.Name,
+            Noun = other.Noun,
+            DisplayName = other.DisplayName,
+            Mysterious = other.Mysterious,
+
+            PacketPrototype = other.PacketPrototype,
+            ProductPrototypes = new List<string>(other.ProductPrototypes),
+            MutationPrototypes = new List<string>(other.MutationPrototypes),
+
+            Chemicals = new Dictionary<string, SeedChemQuantity>(Chemicals),
+            ConsumeGasses = new Dictionary<Gas, float>(ConsumeGasses),
+            ExudeGasses = new Dictionary<Gas, float>(ExudeGasses),
+
+            NutrientConsumption = NutrientConsumption,
+            WaterConsumption = WaterConsumption,
+            IdealHeat = IdealHeat,
+            HeatTolerance = HeatTolerance,
+            IdealLight = IdealLight,
+            LightTolerance = LightTolerance,
+            ToxinsTolerance = ToxinsTolerance,
+            LowPressureTolerance = LowPressureTolerance,
+            HighPressureTolerance = HighPressureTolerance,
+            PestTolerance = PestTolerance,
+            WeedTolerance = WeedTolerance,
+
+            Endurance = Endurance,
+            Yield = Yield,
+            Lifespan = Lifespan,
+            Maturation = Maturation,
+            Production = Production,
+            GrowthStages = other.GrowthStages,
+            HarvestRepeat = HarvestRepeat,
+            Potency = Potency,
+
+            Seedless = Seedless,
+            Viable = Viable,
+            Slip = Slip,
+            Sentient = Sentient,
+            Ligneous = Ligneous,
+
+            PlantRsi = other.PlantRsi,
+            PlantIconState = other.PlantIconState,
+            Bioluminescent = Bioluminescent,
+            CanScream = CanScream,
+            TurnIntoKudzu = TurnIntoKudzu,
+            BioluminescentColor = BioluminescentColor,
+            SplatPrototype = other.SplatPrototype,
+
+            // Newly cloned seed is unique. No need to unnecessarily clone if repeatedly modified.
+            Unique = true,
+        };
+
+        // Adding the new chemicals from the new species.
+        foreach (var otherChem in other.Chemicals)
+        {
+            Chemicals.TryAdd(otherChem.Key, otherChem.Value);
+        }
+
+        // Removing the inherent chemicals from the old species. Leaving mutated/crossbread ones intact.
+        foreach (var originalChem in Chemicals)
+        {
+            if (!other.Chemicals.ContainsKey(originalChem.Key) && originalChem.Value.Inherent)
+            {
+                Chemicals.Remove(originalChem.Key);
+            }
+        }
```

Comment:
I think this is where the issue could be, since we remove inherent chems AFTER adding them, so the inherent chems of the new might species also get cut

Reply:
Disregard that, I misread something

Reply:
I think I found it for real this time